coretex/coretex/dataset/sequence_dataset/sequence_dataset.py

```diff
@@ -50,6 +55,33 @@ def onDecode(self) -> None:
             if sample.id != self.metadata.id
         ]
 
+    @classmethod
+    def createSequenceDataset(
+        cls,
+        name: str,
+        spaceId: int,
+        metadataPath: Union[Path, str],
+        meta: Optional[Dict[str, Any]] = None
+    ) -> Optional[Self]:
+
+        if isinstance(metadataPath, str):
+            metadataPath = Path(metadataPath)
+
+        dataset = CustomDataset.createDataset(name, spaceId, meta)
+        if dataset is None:
+            return None
+
+        if CustomSample.createCustomSample(
+            "_metadata",
+            dataset.id,
+            metadataPath
+        ) is None:
+
+            logging.warning(">> [Coretex] Failed to create _metadata sample")
```

Comment:
When logging in coretexpylib use: `logging.getLogger("coretexpylib").warning(...)`